data/json/item_actions.json

```diff
@@ -244,6 +244,11 @@
     "id": "saw_barrel",
     "name": { "str": "Saw down barrel" }
   },
+  {
+    "type": "item_action",
+    "id": "saw_stock",
+    "name": { "str": "Saw down stock" }
```

Comment:
I think this should specify wooden stock so players are somewhat less likely to waste a key press trying it on the invalid stock types